offerzen/global/employers/employers@4.0.js

```diff
@@ -356,6 +323,7 @@ window.$loaded(function (window, document, $, undefined) {
   })
 
   let skillList = []
+  $('.js-skills-search').attr('autocomplete', 'off')
```

Comment:
I think Mike was getting so annoyed by this one

Reply:
Yess he was freaking annoyed!